events/discordEvents/client/ready.js

```diff
@@ -834,6 +834,52 @@ module.exports = {
 
             sendToChannelFeeds('postUpdate', data, client);
          });
+
+         // =============================================================
+         // Lil Nouns
+         // =============================================================
+         
+         Nouns.on('ProposalCreated', async data => {
```

Comment:
This will listen to events on the main Nouns DAO contract. For LilNouns events, it would be better to use the new LilNouns wrapper.
```
const lilNouns = new nerman.LilNouns(Nouns.provider);
```
Then listen to the event using the wrapper.
```suggestion
         lilNouns.on('ProposalCreated', async data => {
```